templates/long.yaml

```diff
@@ -38,7 +38,14 @@
   can_public_join: false
   description: Test team description
   invitations:
-  - message: Welcome to the Test Team! 
+  - message: Welcome to the Test Team!
     members:
     - principal_id: 3426116
     - email: synapseformation-test-user@sagebase.org
+
+- name: Test Entity View
```

Comment:
This should be under a project as a children.

Reply:
Okay, I amended my commit.